Argument type warnings on ubuntu 18.04

```
  CXX      test/test_btcdeb-utils.o
  CXX      test/test_btcdeb-value.o
  CXXLD    test-btcdeb
  CXX      tap-instance.o
  CXX      tap-functions.o
  CXX      tap-tap.o
tap.cpp: In function ‘int main(int, char* const*)’:
tap.cpp:25:47: warning: format ‘%lld’ expects argument of type ‘long long int’, but argument 3 has type ‘int64_t {aka long int}’ [-Wformat=]
 #define abort(msg...) do { fprintf(stderr, msg); fputc('\n', stderr); exit(1); } while(0)
                                               ^
tap.cpp:371:111:
             abort("pubkey mismatch: input transaction's vout[%lld].scriptPubKey = %s, but our pubkey = %s\n", instance.txin_vout_index, HEXC(spk), HE
XC(serialized_pk));
                                                                                                               ~~~~~~~~~~
tap.cpp:371:13: note: in expansion of macro ‘abort’
             abort("pubkey mismatch: input transaction's vout[%lld].scriptPubKey = %s, but our pubkey = %s\n", instance.txin_vout_index, HEXC(spk), HE
XC(serialized_pk));
             ^~~~~
tap.cpp:25:47: warning: format ‘%lld’ expects argument of type ‘long long int’, but argument 3 has type ‘int64_t {aka long int}’ [-Wformat=]
 #define abort(msg...) do { fprintf(stderr, msg); fputc('\n', stderr); exit(1); } while(0)
                                               ^
tap.cpp:375:125:
             abort("pubkey mismatch: input transaction's vout[%lld].scriptPubKey %s does not end (%s) with our pubkey %s\n", instance.txin_vout_index,
 HEXC(spk), HEXC(cmp), HEXC(serialized_pk));
                                                                                                                             ~~~~~~~~~~
tap.cpp:375:13: note: in expansion of macro ‘abort’
             abort("pubkey mismatch: input transaction's vout[%lld].scriptPubKey %s does not end (%s) with our pubkey %s\n", instance.txin_vout_index$
 HEXC(spk), HEXC(cmp), HEXC(serialized_pk));
             ^~~~~
  CXXLD    tap

```

--- a/tap.cpp
+++ b/tap.cpp
@@ -368,11 +368,11 @@ int main(int argc, char* const* argv)
     if (have_txs) {
         auto spk = instance.txin->vout[instance.txin_vout_index].scriptPubKey;
         if (spk.size() < serialized_pk.size()) {
-            abort("pubkey mismatch: input transaction's vout[%lld].scriptPubKey = %s, but our pubkey = %s\n", instance.txin_vout_index, HEXC(spk), HEXC(serialized_pk));
+            abort("pubkey mismatch: input transaction's vout[%" PRId64 "].scriptPubKey = %s, but our pubkey = %s\n", instance.txin_vout_index, HEXC(spk), HEXC(serialized_pk));
         }
         auto cmp = Item(spk.end() - serialized_pk.size(), spk.end());
         if (cmp != serialized_pk) {
-            abort("pubkey mismatch: input transaction's vout[%lld].scriptPubKey %s does not end (%s) with our pubkey %s\n", instance.txin_vout_index, HEXC(spk), HEXC(cmp), HEXC(serialized_pk));
+            abort("pubkey mismatch: input transaction's vout[%" PRId64 "].scriptPubKey %s does not end (%s) with our pubkey %s\n", instance.txin_vout_index, HEXC(spk), HEXC(cmp), HEXC(serialized_pk));
         }
         btc_logf("Pubkey matches the scriptPubKey of the input transaction's output #%lld\n", instance.txin_vout_index);
     }

